1.8/s2i/bin/assemble

```diff
@@ -6,6 +6,12 @@ pushd /tmp/src
 
 if [[ `go list -f {{.Incomplete}}` == "true" ]]; then
 
+    if [[ -z "$INSTALL_URL" ]]; then
+
+        INSTALL_URL=$IMPORT_URL
```

Comment:
How about doing `INSTALL_URL=${INSTALL_URL:-IMPORT_URL}` instead of the if .. then conditional?

Reply:
Yep, it's better. I've changed it now.